### ISSUE TITLE
Investment Layer 

### ISSUE THREAD

**Issue by IanJakus** [Siqister/cartodb_test_app#36]
Im not trying to add work but looking at the map now I think investment points need a separate layer because its just too cluttered.  What do you think?

**Comment by allthesignals** [comment]
@IanJakus Do you mean they should be split out into a separate toggle-able layer? Can you explain a little more? I think the clustering is helping with the clutter. If there's still too much clutter, I can make the clustering more sensitive to the higher zoom levels (it will cluster at more zoom levels to prevent collisions).

**Comment by IanJakus** [comment]
I don't think that it is clutter, as much as confusing because the investment points are altogether different from the use types. If we had an investment layer like the others such as ownership - that would also explain the difference between public and private investment icons. @allthesignals

**Comment by allthesignals** [comment]
@IanJakus I think I understand - let me try to figure out the data behind that. It should be straightforward to add another layer. @nupoormonani what do you think?
